XenModel/FriendlyNames.resx

```diff
@@ -1638,10 +1638,10 @@
     <value>Help on Licensing</value>
   </data>
   <data name="Label-host.edition-desktop" xml:space="preserve">
-    <value>XenApp/XenDesktop</value>
+    <value>[Citrix] Virtual Apps/Desktops</value>

```

Comment:
I think this should be "[Citrix] Virtual Apps and Desktops". Same for the other occurences.